sql/log.h

```diff
@@ -1201,6 +1201,7 @@ class Log_event_handler
                         const char *user_host, size_t user_host_len, ulonglong query_utime,
                         ulonglong lock_utime, bool is_command,
                         const char *sql_text, size_t sql_text_len)= 0;
+  ATTRIBUTE_FORMAT(printf, 3, 0)
```

Comment:
There isn’t a [guideline](https://github.com/MariaDB/server/blob/-/CODING_STANDARDS.md) on where to put attributes.
I checked existing entries: most put attributes after the signature, a few put them before.